src/components/Input/NormalButtonInput.tsx

```diff
@@ -0,0 +1,157 @@
+'use client';
+
+import { useState } from 'react';
+import { type NormalButtonInputTypes } from '@/components/Input/Input.types';
+import { css } from '@/styled-system/css';
+import { getBorderColor } from '@/utils/getBorderColor';
+
+import Button from '../Button/Button';
+
+export default function NormalButtonInput({
+  value,
+  onChange,
+  errorMsg,
+  positiveMsg,
+  required = false,
+  ...props
+}: NormalButtonInputTypes) {
+  const [isFocused, setIsFocused] = useState(false);
+
+  const statusColor = errorMsg ? 'red.red500' : 'text.tertiary';
+
+  const [doubleCheckabled, setDoubleCheckAbled] = useState(false);
```

Comment:
`doubleCheckabled`를 state로 관리하지 않아도 같은 동작을 수행시킬 수 있을 것 같아요!
`setDoubleCheckAbled` 하는 부분은 삭제해도 될것 같습니다. 
```suggestion
 const doubleCheckabled = inputValue.length === 0;
```

Reply:
disabled 시 brightness 값 확인하려고 넣어 본 상태였는데 제거하지 않았네요 🥲
감사합니다! 6c77b2339698dbcf23d7e2a394a78537af224d2b 에서 제거했습니다.